sources/packages/backend/apps/api/src/route-controllers/student-appeal/student-appeal.institutions.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Param, Get, ParseIntPipe } from "@nestjs/common";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { 
+  AllowAuthorizedParty,
+  IsBCPublicInstitution,
+  HasStudentDataAccess, 
+} from "../../auth/decorators";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import BaseController from "../BaseController";
+import { ClientTypeBaseRoute } from "../../types";
+import { DetailedStudentAppealAPIOutDTO } from "./models/student-appeal.dto";
+import { StudentAppealControllerService } from "./student-appeal.controller.service";
```

Comment:
Can it be added to the index?

Reply:
it is already added, due to the error, that we discussed, i was not able to import from `..`

Reply:
Just double checking, even moving it up or down in the index was not easy to find the proper order and that is why we are using the absolute path, right?

Reply:
yes , you are correct